services/authService.js

```diff
@@ -0,0 +1,42 @@
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+
+const userDao = require("../models/authDao");
+const { validateEmailPw } = require("../utils/validation");
+
+const signUp = async (email, password, name) => {
+    
+    validateEmailPw(email, password);
+
+  const user = await userDao.getUserByEmail(email);
+
+  if (user) {
+    const err = new Error("DUPLICATE_EMAIL!");
+    err.statusCode = 409;
+    throw err;
+  }
+  const hashedPassword = await bcrypt.hash(password, 10);
+  await userDao.createUser(email, hashedPassword, name);
+};
+
+const signIn = async (email, password) => {
+  const user = await userDao.getUserByEmail(email);
+
+  if (!user) {
+    const err = new Error("SPECIFIED_USER_DOES_NOT_EXIST");
+    err.statusCode = 404;
+    throw err;
+  }
+
+  const result = await bcrypt.compare(password, user.password);
+
+  if (!result) {
+    const err = new Error("INVAILD_PASSWORD");
+    err.statusCode = 400;
```

Comment:
```suggestion
    err.statusCode = 401;
```
인증 실패의 경우 `401 status code`를 반환해 주세요!